baiji/copy.py

```diff
@@ -333,7 +359,7 @@ def upload_multipart(self):
     def upload_direct(self):
         import math
         from baiji.util.with_progressbar import FileTransferProgressbar
-        key = self.dst.create()
+        key = self.dst.create(version_id=self.version)
```

Comment:
ah yes, I'll test and remove it. The API makes it look like you can use your own versionID, but probably not.